testing/e2e/workload_test.go

```diff
@@ -421,6 +441,82 @@ func TestCreateFromGitWithAnnotations(t *testing.T) {
 					t.FailNow()
 				}
 			},
+			CleanUp: func(ctx context.Context, t *testing.T) error {
+				ctx = context.WithValue(ctx, "reader", nil)
```

Comment:
is it necessary to set values to `nil` as `ctx` is not return?

Reply:
You're right, I thought I needed to clean up the context every time but it seems it's not necessary. Thanks